Extensions/PlayerAuthentication/playerauthenticationtools.ts

```diff
@@ -171,7 +171,7 @@ namespace gdjs {
       if (!_checkedLocalStorage) {
         readAuthenticatedUserFromLocalStorage();
       }
-      return _userId || null;
+      return _userId || '';
```

Comment:
I've checked for references to the function, and it was used like
```js
const playerId = getUserId();
if(!playerId) // ...
```
So this should be a safe change